lib/uploads_cli.rb

```
@@ -62,6 +63,44 @@
+    template_content = File.read(filename)
+
+    message = {
+      action: 'set_template',
+      key: key,
+      template: template_content
+    }
+    # Send the message to the /tmp/hawksi.sock socket
+    socket = UNIXSocket.new(@socket_path)
+    socket.write(message.to_json)
+    socket.close
+  end
```

rubocop

[convention] Method has too many lines. [14/10]

(Metrics/MethodLength)